docs/architecture/adr-112-proposer-based-timestamps.md

```diff
@@ -318,7 +345,7 @@ By removing fields from the `Precommit` messages, we are able to aggregate signa
 
 * Alters CometBFT’s liveness properties.
 Liveness now requires that all correct validators have synchronized clocks within a bound.
-Liveness will now also require that validators’ clocks move forward, which was not required under `BFTTime`.
+Liveness will now also require that validators’ clocks move forward, which was not required under `BFT Time`.
```

Comment:
Above, the first bullet of positive, I'm not sure I agree with the wording of that. Since byzantine validators (which will eventually, and infinitely often, become proposers) can influence the time within the limits of MSGDELAY and PRECISION

Reply:
Added `arbitrarily` to the sentence.